src/dmd/dsymbolsem.d

```diff
@@ -1036,6 +1036,29 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
             }
         }
 
+        if (dsym.storage_class & STC.mutable)
+        {
+            if (!dsym.isField())
+            {
+                dsym.error("only fields can be `__mutable`");
```

Comment:
This results in hard to understand error message:

```
Error: variable `__mutable2.foo.p` only fields can be `__mutable`
```

maybe "cannot be __mutable. Only fields can."?

Reply:
Or maybe "cannot be declared `__mutable`. `__mutable` can only be applied to fields."